src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -1552,6 +1558,30 @@ public String getModulePropertyValue(ModulePropertyValue property)
         return (String)ref.getValue();
     }
 
+    public void setResponseServerConfigurations(LinkedHashMap<String, String> props)
+    {
+        log("setting response server configuration");
+        goToResponseServerConfiguration();
+
+        for (String prop: props.keySet())
+        {
+            String val = props.get(prop);
+            log("setting property: " + prop + " to value: " + val);
+
+            if (prop.equals("metadataLoadLocation"))
+                checkRadioButton(Locator.radioButtonByNameAndValue("metadataLoadLocation", val));
+            else
+                setFormElement(Locator.name(prop), val);
+        }
+        clickButton("Save and Finish", 0);
```

Comment:
Does "Save and Finish" not navigate? Failing to wait for page loads that occur is a frequent source of intermittent failures.
```suggestion
        clickButton("Save and Finish");
```